File: modules/algorithms-and-ds/modules/data-structures/linear/linked-list/src/cpp/singly-linked-list/driver_printListFromNodeN.cpp
#include "SinglyLinkedList.h"
#include <iostream>

int main()
{
    SinglyLinkedList list;

    list.printListFromNodeN(list.head);

    list.head = new Node(10);
    list.head->next = new Node(20);
    list.head->next->next = new Node(30);

    list.printListFromNodeN(list.head);
    list.printListFromNodeN(list.head->next);
    list.printListFromNodeN(list.head->next->next);

    return 0;
}

File: modules/old-studies/old-algorithms-and-ds/modules/data-structures/linear/lists/src/python/singly-linked-list/driver_printListFromNodeN.py
from SinglyLinkedList import SinglyLinkedList
from Node import Node

print("########## ( Print values from Node 'n' ) ##########")

list = SinglyLinkedList()

list.printListFromNodeN(list.head)

list.head = Node(10)
list.head.next = Node(20)
list.head.next.next = Node(30)

list.printListFromNodeN(list.head)
list.printListFromNodeN(list.head.next)
list.printListFromNodeN(list.head.next.next)
